app/uk/gov/hmrc/voapropertylinking/models/modernised/agentrepresentation/AssignAgent.scala

```diff
@@ -18,8 +18,8 @@ package uk.gov.hmrc.voapropertylinking.models.modernised.agentrepresentation
 
 import play.api.libs.json.{Json, OFormat}
 
-case class AppointAgent(agentRepresentativeCode: Long, scope: String)
```

Comment:
I swear you rename this file every PR.

Reply:
nope. This is the very first time. I renamed something with a similar name in the frontend. Maybe thats what you're thinking of